modules/page.py

```diff
@@ -24,8 +24,8 @@ def add_page(editor):
         add_section(editor)
         editor.autosaver.onChangeMade()
         
-        cmd = Undo({'type':'page','name':title, 'action':'create'})
-        editor.undo_stack.append(cmd)
+        # cmd = Undo({'type':'page','name':title, 'action':'create'})
```

Comment:
can u take these commented lines here and in the other places out if we dont need them

Reply:
Done